solutions/platinum/bts-hotcold.mdx

```diff
@@ -99,11 +90,11 @@ class Tree {
 		tour(1, 0);
 	}
 
-	bool is_ancestor(int u, int v) const {
+	bool is_ancestor(int u, int v) {
```

Comment:
did you uh revert my changes :skull:

Reply:
wait what

uh that was an accident, lemme fix that